src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicencePolicyServiceTest.kt

```diff
@@ -1,29 +1,20 @@
 package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
 
-import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
-import com.fasterxml.jackson.module.kotlin.readValue
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
-import org.springframework.core.io.ClassPathResource
 
 class LicencePolicyServiceTest() {
 
-  private val licencePolicyService = LicencePolicyService(
-    listOf(
-      ClassPathResource("/test_data/policy_conditions/policyV1.json").file,
-      ClassPathResource("/test_data/policy_conditions/policyV2.json").file
-    ).map { policy -> jacksonObjectMapper().readValue(policy) }
-  )
+  private val licencePolicyService = LicencePolicyService()
 
   @Test
   fun `Current policy version is returned`() {
     val policy = licencePolicyService.currentPolicy()
-    assertThat(policy.version).isEqualTo("2.0")
+    assertThat(policy.version).isEqualTo("2.1")
```

Comment:
these tests are a bit noddy but left in to reduce the friction if we extend this area